From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rostyslav Lesovyi <rostyslav.lesovyi@globallogic.com>
Date: Fri, 25 Sep 2026 14:36:05 +0200
Subject: [PATCH] Remove _PLATFORM_TURRIS_ flags

Removed the Turris ETHWAN_DEF_INTF_NAME ("eth2") and a flag from the
wan_physical_ifname syscfg guards.

---
 source/TR-181/board_sbapi/cosa_ethernet_apis.c    | 4 +---
 source/TR-181/board_sbapi/cosa_ethernet_manager.c | 2 +-
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/source/TR-181/board_sbapi/cosa_ethernet_apis.c b/source/TR-181/board_sbapi/cosa_ethernet_apis.c
index a86a09f..9976ea6 100755
--- a/source/TR-181/board_sbapi/cosa_ethernet_apis.c
+++ b/source/TR-181/board_sbapi/cosa_ethernet_apis.c
@@ -665,8 +665,6 @@ COSA_DML_IF_STATUS getIfStatus(const PUCHAR name, struct ifreq *pIfr)
 #define ETHWAN_DEF_INTF_NAME "eth4"
 #elif defined (INTEL_PUMA7)
 #define ETHWAN_DEF_INTF_NAME "nsgmii0"
-#elif defined (_PLATFORM_TURRIS_)
-#define ETHWAN_DEF_INTF_NAME "eth2"
 #elif defined (_COSA_QCA_ARM_) && defined(_PLATFORM_IPQ807x)
 #define ETHWAN_DEF_INTF_NAME "eth4"
 #elif defined (_COSA_QCA_ARM_) && defined (_PLATFORM_IPQ95xx)
@@ -3464,7 +3462,7 @@ CosaDmlEthInit(
         }
     }
 #else
-    #if defined(_PLATFORM_RASPBERRYPI_) || defined(_PLATFORM_TURRIS_) || defined(_PLATFORM_BANANAPI_R4_)
+    #if defined(_PLATFORM_RASPBERRYPI_) || defined(_PLATFORM_BANANAPI_R4_)
 
     char wanPhyName[20] = {0},out_value[20] = {0};
 
diff --git a/source/TR-181/board_sbapi/cosa_ethernet_manager.c b/source/TR-181/board_sbapi/cosa_ethernet_manager.c
index fe97d7f..f6b475b 100644
--- a/source/TR-181/board_sbapi/cosa_ethernet_manager.c
+++ b/source/TR-181/board_sbapi/cosa_ethernet_manager.c
@@ -375,7 +375,7 @@ static ethSmState_t Transition_EthWanLinkFound(PETH_SM_PRIVATE_INFO pstInfo)
 #if defined(FEATURE_RDKB_WAN_AGENT)
     if (ANSC_STATUS_SUCCESS != CosaDmlEthCreateEthLink(pstInfo->Name, stGlobalInfo.Path))
 #elif defined(FEATURE_RDKB_WAN_MANAGER)
-    #if defined(_PLATFORM_RASPBERRYPI_) || defined(_PLATFORM_TURRIS_) || defined(_PLATFORM_BANANAPI_R4_)
+    #if defined(_PLATFORM_RASPBERRYPI_) || defined(_PLATFORM_BANANAPI_R4_)
     CHAR wanPhyName[20] = {0},out_value[20] = {0};
     if (!syscfg_get(NULL, "wan_physical_ifname", out_value, sizeof(out_value)))
     {
